tests/cli/output/p.txt

```diff
@@ -8,72 +8,72 @@ newlines                        = crlf
 # language                      = CPP
```

Comment:
The file p.txt should NOT be part of the PR

Reply:
Ok, so I updated this file because I added PCF_IN_DECLTYPE to the bitfields in pcf_flag_e. If I don't update this, I fail the CLI options test. Please advise.

Reply:
run uncrustify/tests/cli/test_cli_options.py
to see what has changed, and review the output/xyz.txt file

Reply:
I've reviewed the results of the diff, and I think the reason for which I updated the p.txt output file still stands. I've inserted the PCF_IN_DECLTYPE flag at bit 16 (17th LSB), and as such the "Flag" field must be updated to reflect this. For example, if you take line 11 of the existing p.txt output file, you'll see that the value under the Flag column is 1000e0001; however, that value is before I inserted the new flag. With the addition of PCF_IN_DECLTYPE at the 16th bit position, the upper 48 bits must be left-shifted by one (the most significant bit is discarded, but it's not used anyway). This means that 1000e0001 must be updated to the value 2001c0001 for line 11.
```
           1    0    0    0    e    0    0    0    1
Before: 0001 0000 0000 0000 1110 0000 0000 0000 0001
           2    0    0    1    c    0    0    0    1
After:  0010 0000 0000 0001 1100 0000 0000 0000 0001
                               ^
                               insertion here at 16th bit position (17th LSB)
```

I've updated the Flag fields where applicable - some Flag fields did not change since no bits higher than the 16th LSB were set in the first place. I hope this makes sense.